build.rs

```diff
@@ -74,27 +81,57 @@ impl Registry {
 					account_type
 				))
 			}
-			if let Some(sig_type) = &account_type.standard_account {
-				match sig_type.as_str() {
-                    "Sr25519"| "Ed25519" | "secp256k1" | "*25519" => {},
-                    _ => {
-                        return Err(format!("Unknown sig type in standardAccount: expected one of Sr25519, Ed25519, secp256k1, *25519: {:?}", account_type))
-                    }
-                }
+			for (name, decimals) in account_type.symbols.iter().zip(&account_type.decimals) {
```

Comment:
We probably should include some check that `symbols.len() == decimals.len()`?

Reply:
Done [a few lines above](https://github.com/paritytech/ss58-registry/pull/82/files/b73c6345f4091b4f1b073d2d30440120ef28158b#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R78).